devstack-install.md

```diff
@@ -0,0 +1,35 @@
+# Installation guide - devstack scenario
```

Comment:
This looks to me like instructions on how to install OpenStack using devstack so please rename this section accordingly.